app/services/notification_service.rb

```diff
@@ -51,7 +51,8 @@ def self.send_for_follows(follows, entity, event_type, event)
 
   def self.notify_followers(entity, event_type, event, reason_filter = nil, exclude_follows = nil)
     follows = entity.follows.where.not(user_id: event.user.id)
-    
+    follows = follows.where(muted: false)
```

Comment:
this prevents notifications from going out to follows marked as 'muted'

Reply:
probably just merge these two lines, especially if we make the `active` scope